components/gitpod-cli/cmd/env.go

```diff
@@ -276,7 +277,7 @@ func parseArgs(args []string, pattern string) ([]*serverapi.UserEnvVarValue, err
 		val = strings.ReplaceAll(val, `\ `, " ")
 
 		if val == "" {
-			return nil, xerrors.Errorf("variable must have a value; use -u to unset a variable")
+			return nil, GpError{Err: xerrors.Errorf("variable must have a value; use -u to unset a variable"), OutCome: utils.Outcome_UserErr, ErrorCode: utils.UserErrorCode_InvalidArguments}
```

Comment:
Beside this case (server side consider empty string as empty), tested all good

<img width="962" alt="image" src="https://github.com/gitpod-io/gitpod/assets/20944364/e6d41613-a831-4110-856e-70c4f695bc55">

Reply:
@mustard-mh I am not sure I need to change anything here, revert this line?

Reply:
We replace `"\ "` with `" "`, and once server receive `"a= "`, it will throw 400 error.

So it'll be good if client (gp-cli) check this case too. But not that necessary since it's an edge case. @akosyakov

Reply:
let's wait for error group then :D